statsample.gemspec

```diff
@@ -85,5 +85,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'minitest', '~> 5.5'
   s.add_development_dependency 'gettext', '~> 3.1'
   s.add_development_dependency 'mocha', '~> 1.1'
-  # s.add_development_dependency 'gsl-nmatrix', '~> 1.17'
+  s.add_development_dependency 'gsl-nmatrix', '~> 1.17'
```

Comment:
I don't want to add gsl-nmatrix as a dependency without a major release. Can you remove this line before I merge this PR?